less/index.less

```diff
@@ -56,4 +56,83 @@ table {
 html, body {
     height: 100%;
 }
+@screen: flex;
+@bg-color: yellow;
+
+body {
+	background-color: whitesmoke;
+}
+header{
+	border: 6px solid red;
+	background-color:@bg-color;
+	margin-bottom: 2%;
+	#navigation {
+		display: flex;
+		margin: 3%;
+		justify-content: space-between;
+	}
+}
+.Introduction {
+	display: @screen;
+	flex-direction: row;
+	align-items: center;
+	margin-right: 15%;
+	img {
+		display: block;
+		margin: 0 auto;
+	p {
+		display: block;
+		text-align: center;
+		font-size: 100%;
+		padding: 2%;
+		font-weight: bold;
+		color: red;
+	}
+}
+.midddle {
+	display: @screen;
+	justify-content: row;
+	.Skills {
+		text-align: center;
+		padding: 12px 0px 12px 0px;
+		margin-bottom: 2%;
+		margin-left: 65%;
+		display: inline-flex;
+		flex-direction: column-reverse;
+	.Work-History {
+		text-align: center;
```

Comment:
Looked at your html and I see that`Skills` and `Work-History` are siblings. But here you have it set up as if `Work-History` is a child of `Skills`. A great tip for this is looking at where each bracket starts and ends.